tokens/eth/0x67ab11058eF23D0a19178f61A050D3c38F81Ae22.json

```diff
@@ -0,0 +1,34 @@
+{
+  "symbol": "SELF",
+  "name": "SELF TOKEN",
+  "type": "ERC20",
+  "address": "0x67ab11058eF23D0a19178f61A050D3c38F81Ae21",
```

Comment:
This address does not match the filename

Reply:
Sorry. It's fixed!

When I copied and pasted the file, the file name was renamed because there was another file with the same name in the same folder. That's why. 😄